### PR TITLE
[9.0][MIG] crm_lead_to_event_registration module

### PR DESCRIPTION
**Pull request by cubells**
- Updated views
- Updated Readme

cc @Tecnativa

### CODE REVIEW

File: crm_lead_to_event_registration/wizard/crm_lead_event_pick.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class WizardModel(models.TransientModel):
+    _name = "crm.lead.event.pick"
+
+    lead_id = fields.Many2one(
```
Comment (pedrobaeza) [inline crm_lead_to_event_registration/wizard/crm_lead_event_pick.py:11]: Test this wizard.

File: crm_lead_to_event_registration/models/crm_lead.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    @api.multi
+    def action_generate_event_registration(self, event):
+        """Generate an event registration."""
+        er = self.env["event.registration"]
+        for s in self:
+            name = (s.contact_name or
+                    s.partner_name or
+                    s.partner_id.name)
+            if not name:
```
Comment (pedrobaeza) [inline crm_lead_to_event_registration/models/crm_lead.py]: Put here `# pragma: no cover` and this is not critical to test

File: crm_lead_to_event_registration/models/crm_lead.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                "phone": s.phone,
+            })
+            # Load data from partner if available
+            s.event_registration_id._onchange_partner()
+
+    @api.multi
+    def action_check_status_confirm_registration(self):
+        """If the opportunity is won/lost, open/cancel registration."""
+        for s in self:
+            if self._track_subtype(['stage_id']) == "crm.mt_lead_won":
```
Comment (pedrobaeza) [inline crm_lead_to_event_registration/models/crm_lead.py:42]: Test this and change it if XML-IDs have changed.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: crm_lead_to_event_registration/wizard/crm_lead_to_opportunity.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    _inherit = 'crm.lead2opportunity.partner'
+
+    event_id = fields.Many2one(
+        comodel_name="event.event", string="Create registration in this event")
+
+    @api.multi
+    def action_apply(self):
+        res = super(CrmLead2OpportunityPartner, self).action_apply()
+        self.ensure_one()
+        if not self.event_id:
```
Comment (pedrobaeza) [inline crm_lead_to_event_registration/wizard/crm_lead_to_opportunity.py]: Put `event_id` as required and you can remove this.

File: crm_lead_to_event_registration/wizard/crm_lead_event_pick.py (hunk 1 above)
Comment (cubells) [reply crm_lead_to_event_registration/wizard/crm_lead_event_pick.py:11]: Done.

File: crm_lead_to_event_registration/models/crm_lead.py (hunk 3 above)
Comment (cubells) [reply crm_lead_to_event_registration/models/crm_lead.py:42]: Done tests.

**Review by pedrobaeza — APPROVED** [review]

**Review by luismontalba — APPROVED** [review]
Functionality tested on runbot

**Pull request merged**